docs/developer/apis/camino-node-apis/keystore.md

```diff
@@ -8,7 +8,7 @@ Every node has a built-in keystore. Clients create users on the keystore, which
 
 _**You should only create a keystore user on a node that you operate, as the node operator has access to your plaintext password.**_
 
-For validation and delegation on main net, you should issue transactions through [the wallet](https://wallet.camino.foundation). That way control keys for your funds won't be stored on the node, which significantly lowers the risk should a computer running a node be compromised.
+For validation and delegation on main net, you should issue transactions through [the wallet](https://wallet.camino.network). That way control keys for your funds won't be stored on the node, which significantly lowers the risk should a computer running a node be compromised.
```

Comment:
we don't use wallet.camino.network. it should be https://suite.camino.network/wallet